docs/GeccoDonorReport.md

```diff
@@ -0,0 +1,21 @@
+# GECCO Donor Report
```

Comment:
Doesn't look GECCO-specific. Any reason not to add a project parameter and rename it to "Donor Report?"

Reply:
Nothing other than nobody has requested it :grinning: I'll update it